scripts/generate-mjml-react.ts

```diff
@@ -132,7 +132,7 @@ function getPropTypeFromMjmlAttributeType(
 }
 
 function buildTypesForComponent(mjmlComponent: IMjmlComponent): string {
-  const { componentName, allowedAttributes } = mjmlComponent;
+  const { componentName, allowedAttributes, endingTag } = mjmlComponent;
```

Comment:
nit: `isEndingTag` might be a better reacty name

Reply:
Yeah I am aligned to rename it. I would prefer `isEndingTag` as well for the extra semantic meaning